src/rdma.c

```diff
@@ -729,7 +729,7 @@ static void connRdmaEventHandler(struct aeEventLoop *el, int fd, void *clientDat
         return;
     }
 
-    /* uplayer should read all */
+    /* up layer should read all */
```

Comment:
I'm guessing. I couldn't find any other instances and there wasn't much context...